modules/virt-upgrade-pathways.adoc

```diff
@@ -11,6 +11,7 @@ The upgrade pathway is different depending on the 2.4.z version of {VirtProductN
 ====
 You must upgrade your {product-title} to 4.8 before upgrading the minor release of {VirtProductName}.
 ====
+//Is this still correct?
```

Comment:
This module is obsolete. I'm going to delete it in my open PR related to upgrade.